Cargo.toml

```diff
@@ -7,11 +7,11 @@ license = "MIT"
 name = "scram"
 readme = "README.md"
 repository = "https://github.com/tomprogrammer/scram"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
Nah it's a breaking change actually since the `rand`'s traits are exposed, will bump to 0.6.0